File: go/wam/5440.go
const WamWebcMinorEventLog = 5440 // channel: regular

type WebcMinorEventLogEvent struct {
	AdditionalDebugContext string `wam:"additional_debug_context"` // field 1
	LogContext string `wam:"log_context"` // field 2
	LogCount int64 `wam:"log_count"` // field 3
	LogReason string `wam:"log_reason"` // field 4
	LogType LOGTYPE `wam:"log_type"` // field 5
}

File: ts/wam/5440.ts
export const WamWebcMinorEventLog = 5440 as const

export interface WebcMinorEventLogEvent {
  /** field 1, wire `additional_debug_context` */
  additionalDebugContext?: string
  /** field 2, wire `log_context` */
  logContext?: string
  /** field 3, wire `log_count` */
  logCount?: number
  /** field 4, wire `log_reason` */
  logReason?: string
  /** field 5, wire `log_type` */
  logType?: LOGTYPE
}